pages/in/embed/_id/index.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="user-info-div">
+    <div class="user-avatar-border">
+      <div class="user-avatar-wrapper">
+        <img class="avatar" :src="avatar" />
+      </div>
+    </div>
+    <div class="user-identity">
+      <div class="support-div">
+        <span class="support-message">Support </span>
+        <nuxt-link :to="getUserPath"
+          target="_blank"
+          class="display-name">{{ displayName }}</nuxt-link>
+      </div>
+      <div class="provided-by-div">by LikeCoin reinvented LikeButton</div>
+    </div>
+    <nuxt-link :to="getUserPath"
+      target="_blank">
+      <img :src="getLikeButtonSrc"
+           alt="like-button"
+           class="like-button"
+           @mouseover="setLikeButtonState('hover')"
+           @mousedown="setLikeButtonState('click')"
+           @mouseup="setLikeButtonState('normal')"
+           @mouseout="setLikeButtonState('normal')"
+           >
+    </nuxt-link>
+  </div>
+</template>
+
+<script>
+import { LIKE_BTN_IMG_URL } from '~/constant/';
+import { apiGetUserMinById } from '@/util/api/api';
+
+export default {
+  name: 'embed',
+  layout: 'embed',
+  data() {
+    return {
+      LIKE_BTN_IMG_URL,
+      id: '',
+      displayName: '',
+      avatar: '',
+      likeButtonState: 'normal',
+    };
+  },
+  asyncData({ params }) {
+    return apiGetUserMinById(params.id)
+      .then((res) => {
+        const {
+          displayName,
+          avatar,
+        } = res.data;
+        return {
+          id: params.id,
+          displayName,
+          avatar,
+        };
+      })
+      .catch(() => {});
+  },
+  head() {
+    return {
+      link: [
+        {
+          rel: 'prefetch',
+          href: LIKE_BTN_IMG_URL.normal,
+        },
+        {
+          rel: 'prefetch',
+          href: LIKE_BTN_IMG_URL.hover,
+        },
+        {
+          rel: 'prefetch',
+          href: LIKE_BTN_IMG_URL.click,
+        },
+      ],
+    };
+  },
+  computed: {
+    getUserPath() {
+      return `/${this.id}`;
+    },
+    getLikeButtonSrc() {
+      return this.LIKE_BTN_IMG_URL[this.likeButtonState];
+    },
+  },
+  methods: {
+    setLikeButtonState(state) {
+      this.likeButtonState = state;
```

Comment:
should directly manipulate :src instead of computed `getUserPath` for better reactive time